tic-tac-toe/src/index.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import * as serviceWorker from './serviceWorker';
+
+function Square({ value, onClick }) {
+
+  return (
+    <button className="square" onClick={onClick}>
+      {value}
+    </button>
+  );
+}
+
+ReactDOM.render(<Square />, document.getElementById("root"));
+
+function Restart({ onClick }) {
+
+  return (
+    <button className="restart" onClick={onClick}>
+      Play again
+    </button>
+  );
+}
+
+function Game() {
+	const [ squares, setSquares ] = useState(Array(9).fill(null));
+	const [ isXNext, setIsXNext ] = useState(true);
+	const nextSymbol = isXNext ? "X" : "O";
```

Comment:
It was wise to place this ternary statement in a variable. 👍 . Since `nextSymbol` is defined, it can be used down on lines 52 and 66.